### DISCUSSION TITLE
How to use Floating-UI as an standard plain javascript file

### DISCUSSION THREAD

**Discussion by Edddy** [floating-ui/floating-ui#1453] [Q&A] [answered]
I have a basic JQuery website, and I want to incorporate floating-ui.

I'm not into modules, npm, react, nothing

Where can I get a basic .js file to use and how can I use it in and old fashioned way? (If it's possible)

**Comment by atomiks** [comment, marked as answer]
Loading the UMD scripts is what you want https://floating-ui.com/docs/getting-started#umd

**Reply by Edddy** [reply]
And from there? How to get this fiddle to work? https://jsfiddle.net/nd4vycgq/5/

**Reply by atomiks** [reply]
https://jsfiddle.net/1duk9mys/

As the docs mention, the exports are available on `window.FloatingUIDOM`, so that's where `computePosition` lives instead of importing it. But, these concepts are extremely common in front-end dev in 2021, I suggest you learn them because you'll have trouble using many libraries if not:)

**Reply by Edddy** [reply]
Thank you very much, I know I should learn and implement these concepts, just not at this time :)
